doc-content/enterprise-only/admin-and-config/maven-settings-configuration-ref.adoc

```diff
@@ -4,9 +4,26 @@
 
 {PRODUCT} uses Maven `settings.xml` file to configure it's Maven execution. You must create and activate a profile in the `settings.xml` file and declare the Maven repositories used by your {PRODUCT} projects.
 
-.Procedure
-. In the `settings.xml` file, declare the repositories that your {PRODUCT} projects use. Usually, this is either the https://maven.repository.redhat.com/ga/[online {PRODUCT} Maven repository] or the 
-{PRODUCT} Maven repository that you download from the Red Hat Customer Portal and any repositories for custom artifacts that you want to use. 
-. Ensure that {CENTRAL} or {KIE_SERVER} is configured to use the `settings.xml` file, for example by specifying the `kie.maven.settings.custom=<path_to_settings.xml>` property.
-
 For information about the Maven `settings.xml` file, see the Apache Maven Project  https://maven.apache.org/settings.html[Setting Reference].
+
+.Procedure
+. In the `settings.xml` file, declare the repositories that your {PRODUCT} projects use. Usually, this is either the https://maven.repository.redhat.com/ga/[online {PRODUCT} Maven repository] or the
+{PRODUCT} Maven repository that you download from the Red Hat Customer Portal and any repositories for custom artifacts that you want to use.
```

Comment:
Should maven be initial capped here?

Reply:
Yes. On the Apache Maven site, they treat Maven as a proper noun, which I think it is anyway. Did you have an argument for not capping?